### PR TITLE
withCopyToClipboard & ButtonCopyToClipboard

### PR DESCRIPTION
**Pull request by okbel**
### This PR adds 
- withCopyToClipboard
- ButtonCopyToClipboard

### CODE REVIEW

**Review by alinhle — APPROVED** [review]
This is fantastic! 👍

File: plugin-api/beta/client/components/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
 export {Slot} from 'coral-framework/components';
+export {Copy} from 'coral-framework/components';
```
Comment (alinhle) [inline plugin-api/beta/client/components/index.js]: This looks orphaned.

**Review by alinhle — CHANGES_REQUESTED** [review]

File: plugin-api/beta/client/components/index.js [hunk 2]
```diff
@@ -1 +1 @@
-export {Slot, Copy} from 'coral-framework/components';
\ No newline at end of file
+export {Slot, Copy} from 'coral-framework/components';
```
Comment (alinhle) [inline plugin-api/beta/client/components/index.js]: What I meant was, that there is no component `Copy` in `coral-framework/components`, what are you trying to export here?

File: plugin-api/beta/client/components/index.js [hunk 3]
```diff
@@ -1 +1 @@
-export {Slot} from 'coral-framework/components';
+export {Slot, Copy} from 'coral-framework/components';
```
Comment (alinhle) [inline plugin-api/beta/client/components/index.js]: What I meant was, that there is no component Copy in coral-framework/components, what are you trying to export here?

Comment (okbel) [reply plugin-api/beta/client/components/index.js]: Ohhhh, that was old. 🤕

**Review by alinhle — APPROVED** [review]

**Pull request merged**